packages/sync-actions/test/product-sync-prices.spec.js

```diff
@@ -217,6 +217,56 @@ describe('Actions', () => {
             validFrom: dateNow,
             validUntil: twoWeeksFromNow,
           },
+          {
+            // set price custom type
```

Comment:
What does this comment mean? I don't see any `custom` in the lines below?

Reply:
regarding these comments, we have two objects `before` and `now`, this actions created cause of the diff between these two, so i comment the same comment on both `before` and `now`

check line `311`, its same price id `888`, i just mention the reason why this comment exist, got me?!

Reply:
thanks for your explanation and makes sense now 👍